example.fi

```diff
@@ -74,3 +74,12 @@ end
 fn unary_minus(a: Int) : Int do
   -a
 end
+
+# Comparison operators
+fn max(a: Int, b: Int) : Int do
```

Comment:
I know the naming of this function (and what it does) is not consistent with the other ones and I can change it, but I wanted to show something more useful. Let me know what you think.